🐛 base entity 관련 warning

## 🐞 버그 설명
테스트 실행 시, 다음과 같은 warning 메세지가 나옵니다.
```
boost/src/main/java/knu/team1/be/boost/entity/BaseEntity.java:36: warning: @SuperBuilder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
    private boolean deleted = false;
```

@SuperBuilder는 빌더를 생성할 때 필드 초기화식을 무시하기 때문이라는데요.
빌더를 사용할 때도 기본값이 적용되도록 다음과 같이 수정하는 것은 어떤가요?
``` java
@SuperBuilder
public class BaseEntity {

    @Builder.Default
    private boolean deleted = false;
}
```

음 근데 빌더를 사용할 때만 발생하는 거라서 빌더를 사용할 가능성이 있는지 확인을 해보면 좋을 거 같습니다.
빌더가 안쓰인다고 하더라도 @Builder.Default 붙여서 더 견고하게 할 수도 있을 거 같구요.

--- a/src/main/java/knu/team1/be/boost/common/entity/BaseEntity.java
+++ b/src/main/java/knu/team1/be/boost/common/entity/BaseEntity.java
@@ -7,6 +7,7 @@ import jakarta.persistence.Id;
 import jakarta.persistence.MappedSuperclass;
 import java.time.LocalDateTime;
 import java.util.UUID;
+import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.experimental.SuperBuilder;
@@ -33,6 +34,7 @@ public abstract class BaseEntity {
     private LocalDateTime updatedAt;
 
     @Column(name = "deleted", nullable = false)
+    @Builder.Default
     private boolean deleted = false;
 
 }

--- a/src/main/java/knu/team1/be/boost/file/entity/File.java
+++ b/src/main/java/knu/team1/be/boost/file/entity/File.java
@@ -17,6 +17,7 @@ import knu.team1.be.boost.file.entity.vo.StorageKey;
 import knu.team1.be.boost.member.entity.Member;
 import knu.team1.be.boost.task.entity.Task;
 import lombok.AccessLevel;
+import lombok.Builder;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.experimental.SuperBuilder;
@@ -50,6 +51,7 @@ public class File extends BaseEntity {
 
     @Enumerated(EnumType.STRING)
     @Column(name = "status", nullable = false)
+    @Builder.Default
     private FileStatus status = FileStatus.PENDING;
 
     @Column(name = "completed_at")

